stories/module-analytics-4.stories.js

```diff
@@ -43,55 +43,39 @@ function SetupWrap( { children } ) {
 }
 
 storiesOf( 'Analytics-4 Module', module )
-	.add( 'PropertySelect (no account selected)', () => {
+	.add( 'Account Property Select', () => {
 		const setupRegistry = ( { dispatch } ) => {
+			dispatch( STORE_NAME ).receiveGetSettings( {} );
 			dispatch( MODULES_ANALYTICS ).receiveGetSettings( {} );
-			dispatch( MODULES_ANALYTICS ).receiveGetAccounts( [] );
-			dispatch( MODULES_ANALYTICS ).finishResolution( 'getAccounts', [] );
-			dispatch( STORE_NAME ).finishResolution( 'getProperties', [] );
-		};
+			dispatch( MODULES_ANALYTICS ).receiveGetExistingTag( null );
 
-		return (
-			<WithTestRegistry callback={ setupRegistry }>
-				<SetupWrap>
-					<div className="googlesitekit-setup-module__inputs">
-						<PropertySelect />
-					</div>
-				</SetupWrap>
-			</WithTestRegistry>
-		);
-	} )
-	.add( 'PropertySelect', () => {
-		const {
-			createProperty,
-			createWebDataStream,
-			properties,
-			webDataStreams,
-		} = fixtures;
-		const accountID = createProperty._accountID;
-		const propertyID = createWebDataStream._propertyID;
+			const accountA = {
+				id: '1000',
+				name: 'Account A',
+			};
```

Comment:
Why does this need to be called `accountA`? It's the only account for this test, right?

```suggestion
			const account = {
				id: '1000',
				name: 'Account A',
			};
```

Same goes for `propertyA` below.

Reply:
Updated.